scripts/build-images

```diff
@@ -11,7 +11,10 @@ awk '{print $1}' << EOF > build/images-core.txt
     ${REGISTRY}/${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}
 EOF
 
-xargs -n1 -t docker image pull --quiet << EOF >> build/images-core.txt
+# use PULL_CMD=echo to print out the images without pulling them
+PULL_CMD=${PULL_CMD:-"docker image pull --quiet"}
```

Comment:
idk if this matters, but having the quotes inside the variable expansion seems weird?
```suggestion
PULL_CMD="${PULL_CMD:-docker image pull --quiet}"
```

Reply:
It doesn't change the output, but I can include it.